web/cobrands/fixmystreet/assets.js

```diff
@@ -631,7 +649,12 @@ fixmystreet.assets = {
             fixmystreet.map.addControl(fixmystreet.assets.controls[i]);
             fixmystreet.assets.controls[i].activate();
         }
-    }
+    },
+
+    check_layer_relevant: function(layer, category) {
+      return layer.asset_category.indexOf(category) != -1 &&
```

Comment:
`indexOf` is only in IE9+. Site seems to work in IE8 fine apart from assets so would be good to not have this if possible. There’s an `OpenLayers.Util.indexOf` function could use.